internal/build/phases_test.go

```diff
@@ -889,21 +888,6 @@ func testPhases(t *testing.T, when spec.G, it spec.S) {
 				configProvider := fakePhaseFactory.NewCalledWithProvider
 				h.AssertSliceContains(t, configProvider.HostConfig().Binds, expectedBind)
 			})
-
-			it("configures the phase with bind mounts", func() {
```

Comment:
Instead of this test, could we add an assertion around the contents of `configProvider.ContainerOperations()`? (Btw, it looks like that getter is not being used otherwise?)

Reply:
You are right that the getter isn't called. I can remove it. :D. As far as testing the getter, I was under the assumption that a getter/setter test isn't valuable based on earlier [comments](https://github.com/buildpacks/pack/pull/715/files#r447937651).

As for testing the `ContainerOperations` being executed, they are tested via integration test by [`phase_test.go`](https://github.com/buildpacks/pack/pull/715/files#diff-5c8e2e3a86224438e607242a69b117aeR137).

Reply:
Ok, after our slack discussion I see what you are saying now:

>  can we test that for the “detect” phase, that we actually call CopyDir  - by inspecting the contents of configProvider.ContainerOperations()

Added a few more tests for this purpose. Given they are functions the check uses reflection.